src/Kunstmaan/AdminBundle/Toolbar/BundleVersionDataCollector.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Kunstmaan\AdminBundle\Toolbar;
+
+use Kunstmaan\AdminBundle\Helper\Toolbar\DataCollectionInterface;
+use Kunstmaan\AdminBundle\Helper\VersionCheck\VersionChecker;
+use Monolog\Logger;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Session\Session;
+use Symfony\Component\HttpKernel\DataCollector\DataCollector;
+
+class BundleVersionDataCollector extends DataCollector implements DataCollectionInterface
+{
+    /** @var VersionChecker */
+    private $versionChecker;
+
+    /** @var Logger */
+    private $logger;
+
+    /** @var Session */
+    private $session;
+
+    public function __construct(VersionChecker $versionChecker, Logger $logger, Session $session)
+    {
+        $this->versionChecker = $versionChecker;
+        $this->logger = $logger;
+        $this->session = $session;
+    }
+
+    public function getTemplate()
+    {
+        return 'KunstmaanAdminBundle:Toolbar:bundles_version.html.twig';
+    }
+
+    public function getAccessRoles()
+    {
+        return ['ROLE_SUPER_ADMIN'];
+    }
+
+    public function collectData()
+    {
+        $bundleVersionData = $this->session->get('bundle_version_data');
+
+        if ($bundleVersionData) {
+            return ['data' => $bundleVersionData];
+        }
+
+        $data = null;
+        try {
+            $data = $this->versionChecker->check();
```

Comment:
It might be nice to mention somewhere in the docs exactly what data is sent to Kunstmaan's servers when executing a version check (https://github.com/Kunstmaan/KunstmaanBundlesCMS/blob/master/src/Kunstmaan/AdminBundle/Helper/VersionCheck/VersionChecker.php#L94). Is all of that data actually necessary?